usr/harvey/lib/profile

```diff
@@ -8,6 +8,7 @@ fn cd { builtin cd $* && awd }  # for acme
 
 switch($service){
 case terminal
+	test ! -d /mnt/plumb && mkdir /mnt/plumb
```

Comment:
Let's keep it simple. /mnt/plumb should be there, so in a later PR I put it there.